### ISSUE TITLE
Extractable nuclear antigen positivity

### ISSUE THREAD

**Issue by tudorgroza** [obophenotype/human-phenotype-ontology#6087]
**Preferred term label:**
Extractable nuclear antigen positivity

**Synonyms**
ENA positive

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
HP:0030057?

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Sorry for the delay! 
Added as HP:0033476
